manifests/init.pp

```diff
@@ -18,4 +19,10 @@
   validate_absolute_path($customplugins)
 
   validate_string($check_warranty_revision)
+
+  validate_string($api_key)
+
+  class { 'dell::warranty':
```

Comment:
I prefer to use absolute class names. Could you use `::dell::warranty` here please?